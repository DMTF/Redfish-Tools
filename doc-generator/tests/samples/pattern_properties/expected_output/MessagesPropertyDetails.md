### ClearsIf:

The defined property values are listed in [Table TBL_nn++](#table_TBL_nn "ClearsIf property values").
This property contains the available OEM specific actions for this resource.

| string | Description |
| --- | --- |
| SameOriginOfCondition | Indicates the message is cleared by the other message(s) listed in the ClearingLogic object, provided the OriginOfCondition for both Events are the same. |
Table: Table TBL_nn: <a name=table_TBL_nn>ClearsIf property values</a>
### ParamTypes:

The defined property values are listed in [Table TBL_nn++](#table_TBL_nn "ParamTypes property values").
The MessageArg types, in order, for the message.

| string | Description |
| --- | --- |
| number | The parameter is a number. |
| string | The parameter is a string. |
Table: Table TBL_nn: <a name=table_TBL_nn>ParamTypes property values</a>