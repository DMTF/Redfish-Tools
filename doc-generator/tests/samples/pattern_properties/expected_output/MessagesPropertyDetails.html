<h4 id="redfish.dmtf.org/schemas/v1/MessageRegistry.json|details|ClearsIf">ClearsIf</h4>
<p>This property contains the available OEM specific actions for this resource.</p>
<table class="enum enum-details">
<thead>
<tr><th>string</th><th>Description</th></tr>
</thead>
<tbody>
<tr><td>SameOriginOfCondition</td><td>Indicates the message is cleared by the other message(s) listed in the ClearingLogic object, provided the OriginOfCondition for both Events are the same.</td></tr>
</tbody></table>

<h4 id="redfish.dmtf.org/schemas/v1/MessageRegistry.json|details|ParamTypes">ParamTypes</h4>
<p>The MessageArg types, in order, for the message.</p>
<table class="enum enum-details">
<thead>
<tr><th>string</th><th>Description</th></tr>
</thead>
<tbody>
<tr><td>number</td><td>The parameter is a number.</td></tr>
<tr><td>string</td><td>The parameter is a string.</td></tr>
</tbody></table>
